src/feature/redux/books/bookSlice.js

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Kindly avoid disabling linters. They are necessary for upholding coding best practices and uniformity.